xip/xipether.c

```diff
@@ -16,13 +16,121 @@
 #include "libnetlink.h"
 #include "xiart.h"
 
+#define IFINDEX_STR_SIZE 8
+#define XIA_LLADDR_LEN 12
+#define APPEND_ZEROES 20
+
 static int usage(void)
 {
 	fprintf(stderr,
-"Usage:	\n");
+"Usage:	xip ether { addinterface | delinterface } dev DEV\n"
```

Comment:
If you use a tab to separate "Usage:" and "xip ether..." on this line, you should do so on the other lines as well, just to be consistent.